packages/lib/cityTimezonesHandler.ts

```diff
@@ -1,9 +1,8 @@
+import { cityMapping as allCities } from "city-timezones";
+
+export type CityTimezones = Awaited<ReturnType<typeof cityTimezonesHandler>>;
+
 export const cityTimezonesHandler = async () => {
-  /**
-   * Lazy loads third party dependency to avoid loading 1.5Mb for ALL tRPC procedures.
-   * Thanks @roae for the tip 🙏
-   **/
-  const allCities = await import("city-timezones").then((mod) => mod.cityMapping);
```

Comment:
was the way this was imported not necessary ?

Reply:
Resolved by Morgan's refactor [chore: split timezones to its own trpc router](https://github.com/calcom/cal.com/pull/13687/commits/a3e2f5b2e502f073a112ba7b48ad2e54ff5af4de)